packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -72,7 +72,7 @@ abstract class ShaderWarmUp {
   /// `flutter screenshot --observatory-uri=<uri> --type=skia`
   /// and analyze it with https://debugger.skia.org.
   /// Alternatively, one may run the app with `flutter run --trace-skia` and
-  /// then examine the GPU thread in the observatory timeline to see which
+  /// then examine the raster thread in the observatory timeline to see which
```

Comment:
Nit: formerly known as the GPU thread

Reply:
I didn't add the "formerly known" parts in the deepest depths of the framework, and instead put them only in the files most likely seen by a beginner/intermediate user.

I'm not set on that decision, though. What do you think?

Reply:
Sounds good to me. Let's omit "formerly known" in those places.